### PR TITLE
[CI] Fixup the repos file for humble and rolling.

### PR DESCRIPTION
**Pull request by destogl**
Needs backport to humble.

### CODE REVIEW

**Comment by bmagyar** [comment]
So is the failure we see on the binary builds related to not having gen parameters lib from source?

**Comment by destogl** [comment]
> So is the failure we see on the binary builds related to not having gen parameters lib from source?

Puh, I would also like to know that… It shouldn't be because it is released...

File: ros2_controllers-not-released.rolling.repos [hunk 1]
```diff
@@ -1,5 +1,6 @@
 repositories:
-  generate_parameter_library:
-    type: git
-    url: https://github.com/picknikrobotics/generate_parameter_library.git
-    version: main
+  ## EXAMPLE DEPENDENCY
+#   <some_ros_package>:
+#     type: git
+#     url: git@github.com:<some_github_namespace>/<some_ros_package>.git
+#     version: master
```
Comment (destogl) [inline ros2_controllers-not-released.rolling.repos:6]:
```suggestion
  generate_parameter_library:
    type: git
    url: https://github.com/picknikrobotics/generate_parameter_library.git
    version: main
```

**Comment by destogl** [comment]
Ok, now it should work and we can continue, since this blocks PIDController and other controllers

**Comment by bmagyar** [comment]
Thanks. Let's investigate that issue... it's a little disturbing but indeed the current way this PR is it's already improving things so this is getting merged

**Review by bmagyar — APPROVED** [review]

**Pull request merged**